sdk/digitaltwins/Azure.DigitalTwins.Core/src/Customized/DigitalTwinsRestClient.cs

```diff
@@ -55,37 +56,23 @@ internal async Task<Response<string>> AddAsync(string id, string twin, CreateDig
                 throw new ArgumentNullException(nameof(twin));
             }
 
-            using DiagnosticScope scope = _clientDiagnostics.CreateScope("DigitalTwinsClient.Add");
-            scope.Start();
-            try
+            using HttpMessage message = CreateAddRequest(id, twin, digitalTwinsAddOptions);
+            await _pipeline.SendAsync(message, cancellationToken).ConfigureAwait(false);
+            switch (message.Response.Status)
             {
-                using HttpMessage message = CreateAddRequest(id, twin, digitalTwinsAddOptions);
-                await _pipeline.SendAsync(message, cancellationToken).ConfigureAwait(false);
-                switch (message.Response.Status)
-                {
-                    case 200:
-                    case 202:
-                        {
-                            string value = default;
-                            using JsonDocument document = await JsonDocument.ParseAsync(message.Response.ContentStream, default, cancellationToken).ConfigureAwait(false);
-                            value = document.RootElement.GetRawText();
-                            return Response.FromValue(value, message.Response);
-                        }
-                    case 201:
-                        return Response.FromValue<string>(null, message.Response);
-
-                    default:
-                        throw await _clientDiagnostics.CreateRequestFailedExceptionAsync(message.Response).ConfigureAwait(false);
-                }
-            }
-            catch (Exception e)
-            {
-                scope.Failed(e);
-                throw;
+                case 200:
+                    {
+                        Stream value = message.ExtractResponseContent();
+                        return Response.FromValue(value, message.Response);
+                    }
+                case 202:
+                    return Response.FromValue<Stream>(null, message.Response);
```

Comment:
null or value?

Reply:
It's value when the response is 200. When it's 202, we set to null because there is not really any value to return.